docs/docs/guides/guidelines.md

```diff
@@ -0,0 +1,229 @@
+  clear answer.
+- Introduce a variety in prompts by using different phrasing, degrees of
+  politeness or amount of context given.
+- Consider the previous replies and prompts that lead up to the current one.
+- Try to build upon the topic and ask a sensible follow-up question when
+  replying to the assistant.
+
+### Don't:
+
+- Write prompts without a clear request.
+- Make requests that override the original purpose of the assistant, i.e.
+  jailbreak the model.
+- Use the same prompt repeatedly.
+- Change the topic of a conversation without prefacing it accordingly when
+  replying to the assistant.
+- Leave typos and grammatical errors in the prompt.
+- Reply in a language different from the one intended for the data set.
+
+## 4. Classifying an assistant reply {#classifying-assistant}
+
+### Do:
+
+- Rate every criteria of each reply, unless it can't be discerned because it is
+  spam or inappropriate.
+- Judge quality based on how well the reply adheres to the guidelines. Factual
+  accuracy and helpfulness are first and foremost.
+- Make sure to read the reply thoroughly.
+- Use the [label explanations](#label-explanation) to determine which labels
+  apply to the reply.
+- Research to make sure whether the reply is factually accurate.
+- Skip a classification if you are unable to determine the validity of reply.
+
+### Don't:
+
+- Judge quality based on personal beliefs. Assuming an opinion was warranted,
+  fulfills the users request and doesn't violate any guidelines, it should not
+  impact the rating of the reply.
+- Skip a label just because the reply is spam. Each label can help the model
+  improve.
+- Rate a reply's ability to answer a question correctly if you aren't certain.
+
+## 5. Classifying an initial prompt or user reply {#classifying-user}
+
+### Do:
+
+- Rate every criteria of each prompt, unless it can't be discerned because it is
+  spam or inappropriate.
+- Judge quality based on how well the prompt adheres to the guidelines.
+- Make sure to read the prompt thoroughly.
+- Use the [label explanations](#label-explanation) to determine which labels
+  apply to the prompt.
+
+### Don't:
+
+- Judge quality based on personal beliefs. The opinion of the user should not
+  impact the rating of the prompt.
+- Skip a label just because the reply is spam. Each label can help the model
+  improve.
+
+## 6. Ranking assistant replies {#ranking-assistant}
+
+### Do:
+
+- Make sure to read every available reply.
+- Think about which reply best satisfies the request of the user.
+- Rank replies based on how well they adhere to the guidelines. Factual accuracy
+  and helpfulness are first and foremost.
+- Penalize replies that fail to provide adequate warnings or caveats.
+- Penalize replies that are difficult to read due to a lack of formatting,
+  capitalization or other errors.
+- Penalize replies if the requested information is obfuscated by superfluous
+  details that make up a large part of the message.
```

Comment:
Maybe add:

A reply along the lines of "I don't know" is better than a reply that is factually incorrect.